src/commands/edit.ts

```diff
@@ -282,22 +282,50 @@ export async function replaceCharacters(
  *
  * @keys `&` (kakoune: normal)
  */
-export function align(
-  _: Context,
-  selections: readonly vscode.Selection[],
+export function align(_: Context, fill: Argument<string> = " ") {
+  return edit((builder, selections) => {
+    const sortedSelections = Selections.sort(Direction.Forward, [...selections]);
+
+    // Group selections by 'column', nth column being nth selections of each line.
+    const selectionByColumn: vscode.Selection[][] = [];
+    let currentLine: number | undefined = undefined;
+    let currentColumn: number = 0;
+    for (const selection of sortedSelections) {
+      if (selection.start.line !== selection.end.line) {
+        throw new Error(
+          "Cannot align selections that are spanning multiple lines",
```

Comment:
Nice to be handling this edge case now! 😅

```suggestion
          "cannot align selections that span multiple lines",
```

To be more consistent with other error messages.